mmv1/products/notebooks/api.yaml

```diff
@@ -211,6 +211,14 @@ objects:
             the same project, but you must have the service account user
             permission to use the instance. If not specified,
             the Compute Engine default service account is used.
+      - !ruby/object:Api::Type::Array
+        name: 'serviceAccountScopes'
```

Comment:
Will this block be returned by the API even if the user has not specified it? This field would currently cause a diff if so.

Reply:
not at this time.